virtcontainers/device/manager/manager.go

```diff
@@ -104,6 +104,17 @@ func (dm *deviceManager) NewDevice(devInfo config.DeviceInfo) (api.Device, error
 	return dev, err
 }
 
+// RemoveDevice creates bundles of devices based on array of DeviceInfo
```

Comment:
@WeiZhang555 This comment needs to be changed.

Reply:
Done